README.md

```diff
@@ -138,21 +172,23 @@ CREATE TABLE persons(
 Write Ruby code that will create an instance of a person...
 
 ```rb
-# Your code goes here...
+me = Person.create(name: "Tim",age: 27)
 ```
 
 ### Question 8
 
 Assuming the `Person` class from the previous question, write Ruby code that will query for any person that is 15 years of age...
 
 ```rb
-# Your code goes here...
+is_15 = Person.where("age = 15")
 ```
 
 ### Question 9
 
 Write a route in Sinatra that will print "Hello world" in the web browser at the following URL: `http://localhost:4567/oh_hello`
 
 ```rb
-# Your code goes here...
+get '/oh_hello' do
+	puts "Hello World"
+end
```

Comment:
:+1: although you wouldn't `puts` here since we want that actual string to appear in the browser